theano/gof/vm.py

```diff
@@ -809,7 +819,8 @@ def make_vm(self, nodes, thunks,
         pre_call_clear = [storage_map[v] for v in self.no_recycling]
 
         if (self.callback is not None or
-                (config.profile and config.profile_memory)):
+                (config.profile and config.profile_memory) or
+                (self.allow_partial_eval and not self.use_cloop)):
```

Comment:
For now, as it isn't implemented in CVM, remove the (and not self.use_cloop). Otherwise, if use_cloop is True, then it won't allow partial evaluation.